search-commons/src/main/java/no/unit/nva/search/common/records/SwsResponse.java

```diff
@@ -47,8 +47,11 @@ public record Hit(
             String _id,
             double _score,
             JsonNode _source,
+            JsonNode inner_hits,
```

Comment:
Should this be included?

Reply:
We could split this out to another task

Reply:
That is, We'll just keep it as is, and when this functionality is needed or requested, we'll implement the rest...